python/orca/tutorial/NCF/tf_resume_train_xshards.py

```diff
@@ -19,16 +19,16 @@
 
 import tensorflow as tf
 
-from utils import *
 from process_xshards import get_feature_cols, get_label_cols
+from utils import *
```

Comment:
Keep the order? First import process, then model, then utils

Reply:
yep, great